### PR TITLE
Added the parsingFlags function to Moment

### PR DESCRIPTION
**Pull request by davidlipo**
http://momentjs.com/docs/#/parsing/is-valid/

### CODE REVIEW

**Comment by galtalmor** [comment]
That's good.
But could be even better if it returned the actual structure instead of any, although I did not see much details in the docs.

File: moment/moment-node.d.ts [hunk 1]
```diff
@@ -346,6 +346,8 @@ declare namespace moment {
         min(date: Moment | string | number | Date | any[]): Moment;
         min(date: string, format: string): Moment;
 
+        parsingFlags(): any;
```
Comment (adidahiya) [inline moment/moment-node.d.ts]: @davidlipo have you tried using this field? can you try and document the structure of the return object?

**Comment by davidlipo** [comment]
This looks like the structure to me!

File: moment/moment-node.d.ts (hunk 1 above)
Comment (davidlipo) [inline moment/moment-node.d.ts]: Done :)

File: moment/moment-node.d.ts [hunk 2]
```diff
@@ -555,6 +557,17 @@ declare namespace moment {
         lts?: string;
     }
 
+    interface MomentParsingFlagsObject {
```
Comment (adidahiya) [inline moment/moment-node.d.ts]: minor: you can omit the `Object` part of this interface name, it's a little redundant

**Comment by rbuckton** [comment]
@davidlipo ping. Can you please address the comments above?

**Comment by galtalmor** [comment]
:+1:

**Comment by horiuchi** [comment]
@davidlipo Thank you for your contribution.
But this pull request is conflicted. please do `rebase master`.

**Review by adidahiya — APPROVED** [review]

**Comment by andy-ms** [comment]
This still needs to be made merge-able.

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR merge conflicts and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**